WinUIGallery/ControlPages/ScrollViewPage.xaml

```diff
@@ -0,0 +1,236 @@
+<!--
+//*********************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+// THIS CODE IS PROVIDED *AS IS* WITHOUT WARRANTY OF
+// ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING ANY
+// IMPLIED WARRANTIES OF FITNESS FOR A PARTICULAR
+// PURPOSE, MERCHANTABILITY, OR NON-INFRINGEMENT.
+//
+//*********************************************************
+-->
+<Page x:Class="AppUIBasics.ControlPages.ScrollViewPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:AppUIBasics">
+    <StackPanel>
+        <local:ControlExample x:Name="Example1" HeaderText="Content inside of a ScrollView.">
```

Comment:
Does this sample allow panning? If so adding instructions might be helpful.

Reply:
Added comment on top.